README.md

```diff
@@ -261,6 +200,44 @@ Array<{
     address: string,
     amount: string,
   }>,
+  withdrawals: Array<{ address: string,
+    amount: string
+  }>,
+  certificates: Array<{|
+    king: 'StakeRegistration',
+    stakeCredential: string,
+  |} | {|
+    king: 'StakeDeregistration',
+    stakeCredential: string,
+  |} | {|
+    king: 'StakeDelegation',
+    stakeCredential: string,
+    poolKeyHash: string,
+  |} | {|
+    king: 'PoolRegistration',
+    poolParams: {|
+      operator: string,
+      vrf_keyhash: string,
+      pledge: string,
+      cost: string,
+      margin: number,
+      reward_account: string,
+      pool_owners: Array<string>,
+      relays: Array<{ ipv4: string, ipv6: string, dnsName: string, dnsSrvName: string, port: String}>,
+      pool_metadata: null | {|
+        url: string,
+        metadata_hash: string,
+      |},
+    |},
+  |} | {|
+    type: 'PoolRetirement',
+    pool_keyhash: string,
+    epoch: number,
+  |} {|
+    type: 'MoveInstantaneousRewardsCert',
+    rewards: Array<string>,
```

Comment:
again, best to comment what format this address uses (if it's even an address. Maybe it's just the key hash or something?)